### PR TITLE
feat-mobile: balance breakdown

### PR DESCRIPTION
**Pull request by begonaalvarezd**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

Depends on https://github.com/iotaledger/firefly/pull/5688, nested PR

## Changelog

```
- feat-mobile: balance breakdown
```

## Relevant Issues

Close https://github.com/iotaledger/firefly/issues/4967

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows
-   **Mobile**
    -   [ ] iOS
    -   [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick all of the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/mobile/views/dashboard/drawers/account-actions/views/BalanceBreakdownView.svelte [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+    $: totalStorageDeposit = accountBalance?.requiredStorageDeposit
+        ? Object.values(accountBalance?.requiredStorageDeposit).reduce(
+              (total: number, value: string): number => total + Number(value),
+              potentiallyLockedOutputsStorageDeposit
+          )
+        : potentiallyLockedOutputsStorageDeposit
+
+    function handleConsolidation(): void {
+        openPopup({
```
Comment (paul-boegelsack) [inline packages/mobile/views/dashboard/drawers/account-actions/views/BalanceBreakdownView.svelte:41]: Reminder to remove the popup logic

**Review by paul-boegelsack — CHANGES_REQUESTED** [review]
lgtm 🔥 I just added a comment because of the popup logic

File: packages/mobile/views/dashboard/drawers/account-actions/views/BalanceBreakdownView.svelte (hunk 1 above)
Comment (begonaalvarezd) [reply packages/mobile/views/dashboard/drawers/account-actions/views/BalanceBreakdownView.svelte]: @paul-boegelsack fixed 🙏🏼

**Review by paul-boegelsack — APPROVED** [review]
🔥

**Pull request merged**